lapixdl/convert.py

```diff
@@ -300,6 +307,136 @@ def __generate_coco_od_file(
     return coco_od
 
 
+# -----------------------------------------------------------------------
+# Functions to work with lapix DataFrame
+def lapix_to_annotations(lapix_df: LapixDataFrame) -> list[Annotation]:
+    return [
+        Annotation(
+            row['geometry'],
+            row['category_id'],
+            row['iscrowd'] if 'iscrowd' in row else 0
+        )
+        for _, row in lapix_df.iterrows()
+    ]
+
+
+# -----------------------------------------------------------------------
+# Functions to generate masks
+def sort_annotations_to_draw(
+    annotations: list[Annotation],
+    draw_order: tuple[int, ...] | None = None
+) -> list[Annotation]:
+    '''Will sort the annotations based on the draw_order, if draw_order
+    is None will sort in ascending order based in category value'''
+    items = defaultdict(list)
+
+    for ann in annotations:
+        items[ann.category_id].append(ann)
+
+    if draw_order is None:
+        draw_order = tuple(np.unique(list(items)))
+    elif not isinstance(draw_order, tuple):
+        raise ValueError('Unexpected value for `draw_order`. Need to be a tuple of int, or None.')
+
+    out = [ann for cat in draw_order for ann in items[cat]]
+
+    return out
+
+
+def draw_annotation(
+    target: Image.Image,
```

Comment:
I don't think this module should deal with images. The mask should be a plain 1D numpy or python array. If it's supposed to be an internal function, use the `__` prefix.

Reply:
This is a PIL image to use the [PIL.ImageDraw.Draw](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.Draw) function... If you have some suggestion how to do this with `numpy` (which will be a C/C++) we can change it